docker/Dockerfile

```diff
@@ -7,18 +7,24 @@ COPY docker/entrypoint.sh /usr/local/bin/
 RUN chmod 755 /usr/local/bin/entrypoint.sh
 
 RUN apk add --no-cache \
-	autoconf \
-	make \
-	g++ \
-	bash \
-	git \
-	openssl-dev
-
-RUN echo -e 'memory_limit=2G' > /usr/local/etc/php/conf.d/memory.ini
+    autoconf \
+    bash \
+    g++ \
+    git \
+    icu-libs \
+    libintl \
+    icu \
+    icu-dev \
+    libxml2-dev \
+    openssl-dev \
+    make
```

Comment:
Please use alpha sort here, it will make it easier to see if a package is already installed, and will make adding new packages result in smaller diffs.

Reply:
Done